postgres/init/20-portal.sql

```diff
@@ -29,15 +29,21 @@ CREATE TABLE portal.company_roles (
 );
 
 
+CREATE TABLE portal.iam_clients (
+    id uuid PRIMARY KEY,
+    client_client_id character varying(255) NOT NULL,
+    CONSTRAINT uk_iam_client_client_client_id UNIQUE(client_client_id)
+);
```

Comment:
"client_client_id" ? Is it a mistake or on purpose?

Reply:
that is on purpose. The table in keycloak-db is named 'clients', the column than is named 'client_id'. (There is also a column 'id' there which contains the technical id aka 'id of client' that we may later decide to include here for efficiency reasons. That column than would be named 'client_id' here. For now but I didn't include this foreign id column here because it may be difficult to keep in sync amongs different keycloak installations)